paddle/fluid/framework/ir/conv_bn_fuse_pass.cc

```diff
@@ -413,7 +414,7 @@ void ConvBNFusePass::ApplyImpl(ir::Graph* graph) const {
     if (is_mkldnn) {
       if (conv->Op()->Type() == "conv2d" ||
           conv->Op()->Type() == "depthwise_conv2d") {
-        conv->Op()->SetType("fused_conv2d");
+        ConvertToFusedOp(conv->Op());
```

Comment:
You can remove `if` statement because this check will be done in `ConvertToFusedOp` function.